xmodule/capa_block.py

```diff
@@ -2196,15 +2288,19 @@ def get_score(self):
 
     def update_correctness(self):
         """
-        Updates correct map of the LCP.
+        Updates the `correct_map_history` and the `correct_map` of the LCP.
+
         Operates by creating a new correctness map based on the current
         state of the LCP, and updating the old correctness map of the LCP.
         """
-        # Make sure that the attempt number is always at least 1 for grading purposes,
-        # even if the number of attempts have been reset and this problem is regraded.
```

Comment:
why are we removing this explanatory comment? It gives a bit more context on 2296

Reply:
Oops, I added it again